epytope/EpitopePrediction/PSSM.py

```diff
@@ -89,19 +89,18 @@ def __load_allele_model(allele, length):
                     continue
 
                 if alleles_string[a] not in result:
-                    result[alleles_string[a]] = {}
+                    result[alleles_string[a]] = {self.name+'Score': {}}
```

Comment:
hardcoded but as discussed we can take care of that in a separate PR, I won't mention it for the remaining occurrences

Reply:
Adressed in #43